README.md

```diff
@@ -1,24 +1,207 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
-Things you may want to cover:
+Modify this file to match your project and remove sections that don't apply.
 
-* Ruby version
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
-* System dependencies
+After you're finished please remove all the comments and instructions!
+-->
 
-* Configuration
+<div align="center">
+  <br/>
 
-* Database creation
+  <h3><b>Hello Rails React API App</b></h3>
 
-* Database initialization
+</div>
 
-* How to run the test suite
+<!-- TABLE OF CONTENTS -->
 
-* Services (job queues, cache servers, search engines, etc.)
+# 📗 Table of Contents
 
-* Deployment instructions
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Hello Rails React API App ](#-hello-rails-react-api-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Ruby on Rails ](#ruby-on-rails-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* ...
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Rails React API App <a name="about-project"></a>
+
+**The Hello Rails React API App** is a combined Rails and React API based web application that returns different kinds of greetings via an API endpoint that is interfaced with a React Front End application that consuming the API endpoints bundled together with WEBPACK.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Ruby on Rails <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **API Endpoints**
+- **Database management with Postgres**
+- **ORM with ActiveRecord**
+- **React Front End**
+- **Bundled with WebPack**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> A live demo link will be added soon.
+
```

Comment:
- [ ] Kindly remove this Live Demo Section since you don't have a live link so it would be nice if you remove this section to make your readme file descriptive and professional 😊 Once you would have your live link you can add this section again with the working live link 👍